app/views/layouts/application.html.erb

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
-<html>
+<html lang='en'>
   <head>
     <title>BudgetingKid</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <%  title = "BudgetingKid - Foster good financial habits in your children"
-        description = <<~description
+    <%  title = @title || "BudgetingKid - Foster good financial habits in your children"
```

Comment:
Can we instead use something like that https://guides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method ?